git_machete/client.py

```diff
@@ -843,7 +847,8 @@ def traverse(
         else:
             msg = f"No successor of {bold(current_branch)} needs to be slid out or synced with upstream branch or remote"
         sys.stdout.write(f"{msg}; nothing left to update\n")
-
+        if not is_action_done and initial_branch != self.managed_branches[0]:
```

Comment:
Actually, as I'm now looking into the original issue #318... AFAICS this message would still NOT be printed in Konrad's case, right? maybe the just the `not is_action_done` part should be left (?)

Reply:
It depends on case, there are several in Konrad's example:
- start from `issue/290-fish-completion` where there is nothing to be done: Tip will be printed.
- start from `develop` where some action actions needs to be done: Tip will not be printed.

And other possible cases:
- start from `develop` but everything is synced: Tip will be printed.
- start from `master` : Tip will not ever be printed, cause we checked all branches from the definition file.

Reply:
hmm, maybe it should not print tip when starts from any of root branch?

Reply:
Okay! let's try _ not print tip when starts from any root branch (not just the **first** root branch)_ approach then